javascripts/discourse/templates/components/top-contributors.hbs

```diff
@@ -10,7 +10,11 @@
         class="top-contributors--user-badge"
       >
         {{avatar item.user imageSize="small"}}
-        {{html-safe item.user.username}}
+        {{#if (and item.user.name siteSettings.prioritize_username_in_ux)}}
```

Comment:
I think the logic is inverted... `if prioritize_username_in_ux` we should render `item.user.username`